openedx/core/djangoapps/user_api/accounts/signals.py

```diff
@@ -6,10 +6,10 @@
 from django.dispatch import Signal
 
 # Signal to retire a user from LMS-initiated mailings (course mailings, etc)
-USER_RETIRE_MAILINGS = Signal(providing_args=["user"])
+USER_RETIRE_MAILINGS = Signal()
```

Comment:
As in the previous PR comments, should we move the argument info to code comments?